indico/modules/events/papers/client/js/components/PaperContent.jsx

```diff
@@ -0,0 +1,66 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2019 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import React from 'react';
+import {useSelector} from 'react-redux';
+
+import {Translate} from 'indico/react/i18n';
+
+import {getPaperDetails} from '../selectors';
+import PaperFiles from './PaperFiles';
+import PaperMetadata from './PaperMetadata';
+
+export default function PaperContent() {
+  const {
+    lastRevision: {spotlightFile, files},
+  } = useSelector(getPaperDetails);
+  const onlySpotlightFile = files.length === 1;
+  let content;
+
+  if (spotlightFile) {
+    content = (
+      <>
+        <div
+          key={`spotlight-file-${spotlightFile.id}`}
+          className={`flexrow f-a-center spotlight-file i-button text-color borderless paper-file ${
+            onlySpotlightFile ? 'full-width' : ''
+          }`}
+          title={spotlightFile.filename}
+        >
+          <a
+            href={spotlightFile.downloadURL}
+            className={`attachment spotlight-file-name flexrow f-a-center truncate-text ${spotlightFile.icon}`}
+          >
+            <span className="title truncate-text">{spotlightFile.filename}</span>
+          </a>
+        </div>
+        {!onlySpotlightFile && (
+          <PaperFiles
+            key="paper-files"
+            files={files.filter(file => file.id !== spotlightFile.id)}
+          />
+        )}
+      </>
+    );
+  } else {
+    content = (
+      <div key="paper-files" className="paper-files-list flexrow f-a-baseline">
+        <div className="paper-files-list-label f-self-no-shrink">
+          <Translate>Paper files</Translate>
```

Comment:
Do we need that? I think it's clear without the title
![image](https://user-images.githubusercontent.com/26142639/68303476-3f9b4b80-00a4-11ea-98d7-850cbe096f59.png)

Reply:
For now I just did it 1:1 to what we had before. I think that along the way we will improve it anyway ;)